### DISCUSSION TITLE
White text on black background possible in 3.5.2?

### DISCUSSION THREAD

**Discussion by DigiAngel** [sumatrapdfreader/sumatrapdf#4531] [General]
Topic.  I can get white on black in 3.4.6, but sadly not in 3.5.2.  Any workaround for this?  On Windows...thank you.

**Comment by GitHubRulesOK** [comment]
Sure it should have inversion press `i` key
![image](https://github.com/user-attachments/assets/bb6bbaa7-8571-42e2-9970-b1269622aa84)


but no way to alter an older version behaviours beyond those existing at that time 3.5.s was a changeling and really for a good set of newer features use current August 3.6  Pre-release where it is SHIFTED `i`

**Comment by DigiAngel** [comment]
That'll work thank you!
